tensorflow/contrib/pi_examples/README.md

```diff
@@ -0,0 +1,39 @@
+# TensorFlow Raspberry Pi Examples
+
+This folder contains examples of how to build applications for the Raspberry Pi using TensorFlow.
+
+## Building the Examples
+
+ - Follow the Raspberry Pi section of the instructions at [tensorflow/contrib/makefile](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/contrib/makefile) to compile a static library containing the core TensorFlow code.
+
+ - Install libjpeg, so we can load image files:
+
+```
+sudo apt-get install -y libjpeg-dev
+```
```

Comment:
If we're only providing Ubuntu examples, call that out: "You'll need to make libjpg and its headers available. On Ubuntu, do [x]".